specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/Sessions.json

```diff
@@ -0,0 +1,374 @@
+            "in": "path",
+            "description": "Name of the Session Pool.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z][a-z0-9]*$",
+            "minLength": 3,
+            "maxLength": 63
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SessionCollection"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Sessions by Session Pool": {
+            "$ref": "./examples/Sessions_ListBySessionPool.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SSHConfiguration": {
+      "description": "SSH configuration of the Session.",
+      "type": "object",
+      "properties": {
+        "username": {
+          "description": "The username.",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password.",
+          "type": "string"
+        },
+        "authorizedPublicKey": {
+          "description": "The authorized public key.",
+          "type": "string"
+        }
+      }
+    },
+    "CodeExecutionConfiguration": {
+      "description": "Code execution configuration of the Session.",
+      "type": "object",
+      "properties": {
+        "token": {
+          "description": "The token used for code execution.",
+          "type": "string"
+        },
+        "environment": {
+          "description": "The environment used for code execution.",
+          "type": "string"
+        }
+      }
+    },
+    "SessionTemplate": {
+      "description": "Session template.",
+      "type": "object",
+      "properties": {
+        "requestedDurationInSeconds": {
+          "description": "The requested duration of the Session in seconds.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "timeElapsedInSeconds": {
+          "description": "The time elapsed in seconds since the Session was created.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "expiryTime": {
+          "description": "The time when the Session will expire.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "generatedTime": {
+          "description": "The time when the Session was generated.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "connectionString": {
+          "description": "The connection string of the Session.",
```

Comment:
Since it is sensitive information then we should not return back in client response.  More information: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-secrets-allowed-in-post-response

Not sure if this is applicable for ARM APIs too

Reply:
Removed this now